cypress/integration/layers/eventlayer.cy.js

```diff
@@ -16,7 +38,24 @@ context('Event Layers', () => {
         cy.contains('Program is required').should('be.visible')
     })
 
-    it('adds an event layer', () => {
+    it('shows error if no endDate is specified', () => {
+        Layer.openDialog('Events')
+            .selectProgram('Inpatient morbidity and mortality')
+            .validateStage('Inpatient morbidity and mortality')
+            .selectTab('Period')
+            .selectPeriodType('Start/end dates')
+            .typeStartDate(`${CURRENT_YEAR - 5}-02-01`)
```

Comment:
With the use of the "-5" in several places I wonder if it might be informative to future us to create a test object, for example:
```
{
programName: 'Inpatient morbitity and mortality',
yearWithData: CURRENT_YEAR - 5
}